engine/src/main/java/org/destinationsol/game/screens/MainGameScreen.java

```diff
@@ -289,12 +294,33 @@ public void updateCustom(SolApplication solApplication, SolInputManager.InputPoi
 
         updateTalk(game);
 
+        if (solApplication.getNuiManager().hasScreen(consoleScreen)) {
+            controls.forEach(x -> x.setEnabled(false));
+            consoleControlGrave.setEnabled(true);
+            consoleControlF1.setEnabled(true);
+        }
+
         if (pauseControl.isJustOff()) {
             game.setPaused(!game.isPaused());
         }
 
+        if (consoleScreen.isConsoleJustClosed()) {
+            game.setPaused(false);
+            controls.forEach(x -> x.setEnabled(true));
+            consoleControlGrave.setEnabled(true);
+            consoleControlF1.setEnabled(true);
+
+            menuControl.update(inputPointers, false, false, inputMan, solApplication);
+            menuControl.update(inputPointers, false, false, inputMan, solApplication);
```

Comment:
I think that was needed when I was testing the NUI-based console early-on but removing it doesn't seem to make any difference so both those lines have now been removed. I believe it used to resolve certain input-focus issues but since the game is now paused when the console is open it isn't needed anymore.